CIP-0035/CIP-0035.md

```diff
@@ -0,0 +1,218 @@
+
+```html
+<meta name="cip-31:authenticate" value="123456789012345678901234567890123456789012345678901234564558414D504C452F50"/>
+```
+
+## Example 2 - On-Chain Reference
+
+In this example, we use the same data, but instead construct the metadata as an off-chain reference.
+
+```json
+{
+  "20": {
+    "12345678901234567890123456789012345678901234567890123456": {
+      "4558414D504C452F50": {
+        "ref": "https://assets.example.com/"
+      }
+    }
+  }
+}
+```
+
+In this example, the `ref` url returns the following JSON encoded content.  Note that this URL also contains data for other tokens, but because we know the policy id and the hex encoded asset name of the `EXP`, we can uniquely identify from this list.
+
+```json
+{
+  "20": {
+    "12345678901234567890123456789012345678901234567890123456": {
+      "4558414D504C452F50": {
+        "ticker": "EXP",
+        "url": "https://example.com/asset.json",
+        "desc": "Example Publisher Token",
+        "icon": "ipfs://ipfs/1234567890123456789012345678901234567890123456",
+        "icon-16": "ipfs://ipfs/1234567890123456789012345678901234567890123456",
+        "decimals": 4
+      }
+    },
+    "12345678901234567890123456789012345678901234567890123456": {
+      "123456789012345678": {
+        "ticker": "OTHER-TOKEN",
+        "url": "https://other-token.com/",
+        "desc": "other token included for example purposes",
+        "icon": "ipfs://ipfs/1234567890123456789012345678901234567890123457",
+        "decimals": 2
+     }
+    }
+  }
+}
+```
+
+Unlike the previous example, this example returns a JSON encoded authenticity data as follows:
+
+```json
+{
+  "20": {
+    "12345678901234567890123456789012345678901234567890123456": {
+      "4558414D504C452F50": {
+        "ticker": "EXP",
+        "url": "https://example.com/asset.json",
+        "desc": "Example Publisher Token",
+        "icon": "ipfs://ipfs/1234567890123456789012345678901234567890123456",
+        "icon-16": "ipfs://ipfs/1234567890123456789012345678901234567890123456",
+        "decimals": 4
+      }
+    },
+    "12345678901234567890123456789012345678901234567890123456": {
+      "123456789012345678": {
+        "ticker": "OTHER-TOKEN",
+        "url": "https://other-token.com/",
+        "desc": "other token included for example purposes",
+        "icon": "ipfs://ipfs/1234567890123456789012345678901234567890123457",
+        "decimals": 2
+     }
+    }
+  }
+}
+```
+
+Notice this data is the same data as returned by the `ref` for the on-chain reference.  This is intentionally designed to reduce the amount of 
+boilerplate required to satisfy this CIP.
+
+## Notes
+
+This CIP is intended to handle the common use faced primarily by exchanges as wallets that need to display tokens to their users.  
+Other standards, like `CIP-25`, the NFT metadata standard, will be more appropriate for handling specialized use cases.  
+
+It is not expected nor desired for all tokens to contain `CIP-31` metadata.
+
+
+
```

Comment:
Any move forward on this CIP?
Is it likely or onlikely it will be supported?
I think positive on On-Chain Token Metadata Standard, becouse currently fungible tokens massively being minted with CIP-25 which is confusing. And offchain token registry is not robust solution

Reply:
If one need an example to try support draft CIP, I have minted some according to this CIP:
https://pool.pm/$whiteman/%40f0a2917d